.rustfmt.toml

```diff
@@ -2,5 +2,3 @@ edition = "2021"
 reorder_imports = true
 hard_tabs = true
 max_width = 120
-comment_width = 100
-wrap_comments = true
```

Comment:
Is this removed because we are not using nightly anymore?

Reply:
yes